core/shared/src/main/scala/zio/Config.scala

```diff
@@ -179,6 +179,8 @@ object Config {
     }
 
     def value: Chunk[Char] = Chunk.fromArray(raw)
+
+    def stringValue = new String(raw)
```

Comment:
```suggestion
    def stringValue = new String(raw, StandardCharsets.UTF_8)
```

Reply:
Thank you for the suggestion @guizmaii, but I don't think it applies. `raw` is not an `Array[Byte]`, it's an `Array[Char]`, so no character decoding is being done here.